docker-compose.yaml

```diff
@@ -12,7 +12,7 @@ services:
     volumes:
       - pgdata:/var/lib/postgresql/data
     ports:
-      - 5432:5432
+      - 15432:5432
```

Comment:
Sneaking this in here to make connecting to the database possible when Postgres is installed on the host machine.